go/adcm/status/host.go

```diff
@@ -76,6 +108,7 @@ func (db dbHost) retrieve(id int) (Host, bool) {
 	return Host{Id: id, MaintenanceMode: false}, false
 }
 
+// update - updated or created Host entity, implements the PUT method
```

Comment:
Noticed one thing now:
this is a method of "low-level" structure and it returns "code" from API level instead of flag "updated/created"

I suggest trying to avoid such a mixup in further work, so we can clearly see what and where is set (fact of creation set where it's updated/created, but response code is detected on API level).

I suggest "update or create" instead of past tence, because it's what this method will do when you call it, not what it has already done when it was defined

Reply:
fixed